arangod/Cluster/FollowerInfo.cpp

```diff
@@ -542,3 +543,29 @@ VPackBuilder FollowerInfo::newShardEntry(VPackSlice oldValue) const {
   }
   return newValue;
 }
+
+uint64_t FollowerInfo::newFollowingTermId(ServerID const& s) noexcept {
+  uint64_t i = 0;
+  // We want the random number to be non-zero:
+  do {
+    i = RandomGenerator::interval(UINT64_MAX);
+  } while (i == 0);
+  try {
+    _followingTermId[s] = i;
```

Comment:
I think this method can be called by multiple threads (for multiple followers) concurrently, so `_followingTermId` needs to be access-protected by either a mutex or an RW-lock.

Reply:
Indeed, will fix.

Reply:
Done.